web/src/components/merchant/sign-up/SignUpCard.tsx

```diff
@@ -118,24 +118,50 @@ const SignUpCard: React.FC = () => {
     // If both passwords are empty we show PASSWORD_EMPTY.
     event.target.value === password
       ? setConfirmPasswordError('')
-      : setConfirmPasswordError(PASSWORDS_DO_NOT_MATCH);
+      : setConfirmPasswordError(Errors.PASSWORDS_DO_NOT_MATCH);
     setConfirmPassword(event.target.value);
   };
 
   const handleVpaChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    event.target.value ? setVpaError('') : setVpaError(VPA_EMPTY);
+    event.target.value ? setVpaError('') : setVpaError(Errors.VPA_EMPTY);
     setVpa(event.target.value);
   };
 
   const handleSubmit = () => {
     // These checks are needed because empty error in the handleChange is shown
     // only if user already fills in the field and then deletes it.
-    if (!name) setNameError(NAME_EMPTY);
-    if (!email) setEmailError(EMAIL_EMPTY);
-    if (!password) setPasswordError(PASSWORD_EMPTY);
+    if (!name) setNameError(Errors.NAME_EMPTY);
+    if (!email) setEmailError(Errors.EMAIL_EMPTY);
+    if (!password) setPasswordError(Errors.PASSWORD_EMPTY);
     if (password !== confirmPassword)
-      setConfirmPasswordError(PASSWORDS_DO_NOT_MATCH);
-    if (!vpa) setVpaError(VPA_EMPTY);
+      setConfirmPasswordError(Errors.PASSWORDS_DO_NOT_MATCH);
+    if (!vpa) setVpaError(Errors.VPA_EMPTY);
+
+    if (!name || !email || !password || password !== confirmPassword || !vpa) {
```

Comment:
Is it possible to avoid doing a second time check here? If there is a growing list of validation checks (e.g., 20 items), this is a long line of repetition.

Reply:
Changed to using library so we don't have to do this.